src/axes/radialAxis.ts

```diff
@@ -0,0 +1,198 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Axes {
+  export class RadialAxis extends Axis<number> {
+
+    /**
+     * Constructs a Radial Axis.
+     *
+     * A Radial Axis is a visual representation of a QuantitativeScale in polar system.
+     *
+     * @constructor
+     * @param {QuantitativeScale} scale
+     * @param {string} orientation One of "left"/"right"
+     */
+    constructor(scale: QuantitativeScale<number>, orientation: string) {
+      super(scale, orientation);
+      this.formatter(Formatters.general());
+      this.tickLabelPadding(5);
+      this.removeClass("x-axis");
+      this.removeClass("y-axis");
+      this.addClass("r-axis");
+    }
+
+    public computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number) {
+      let range = this._scale.range();
+      super.computeLayout(origin, availableWidth, availableHeight);
+      this._scale.range(range);
+      this.content().attr("transform", "translate(" + this.width() / 2 + "," + this.height() / 2 + ")");
+      return this;
+    }
+
+    public fixedHeight() {
+      return false;
+    }
+
+    public fixedWidth() {
+      return false;
+    }
+
+    public renderImmediately() {
+      super.renderImmediately();
+      let tickMarkValues = this._getTickValues();
+
+      let tickLabelTextAnchor = this.orientation() === "left" ? "start" : "end";
+      let tickLabels = this._tickLabelContainer
+                           .selectAll("." + Axis.TICK_LABEL_CLASS)
+                           .data(tickMarkValues);
+      tickLabels.enter().append("text").classed(Axis.TICK_LABEL_CLASS, true);
+      tickLabels.exit().remove();
+
+      tickLabels.style("text-anchor", tickLabelTextAnchor)
+                .style("visibility", "inherit")
+                .attr(this._generateTickLabelAttrHash())
+                .text((s: any) => this.formatter()(s));
+
+      if (!this.showEndTickLabels()) {
+        this._hideEndTickLabels();
+      }
+      this._hideOverlappingTickLabels();
+      this._hideOverflowingTickLabels();
+      return this;
+    }
+
+    public orientation(orientation?: string): any {
+    if (orientation == null) {
+      return this._orientation;
+    } else {
+      let newOrientationLC = orientation.toLowerCase();
+      if (newOrientationLC !== "left" &&
+          newOrientationLC !== "right") {
+        throw new Error("unsupported orientation");
+      }
+      this._orientation = newOrientationLC;
+      this.redraw();
+      return this;
+    }
+  }
+
+    protected _generateTickLabelAttrHash() {
```

Comment:
Should be `private`, since this has no subclasses yet and it's not overrriding a parent method.